### ISSUE TITLE
Dependencies are misformatted

### ISSUE THREAD

**Issue by r00ster91** [tamasfe/taplo#124]
I'm using this as a VS Code extension and this is how taplo formats part of my Cargo.toml when I save:
```rs
[dependencies]
random = { package="rand", version="0.8.4" }
```
It removes the space before and after `=`.
I'm using v0.12.3.

**Comment by tamasfe** [comment]
Sorry about the late reply, the formatting code has become unmaintainable and I'm currently working on an overhaul. For those who have this issue, you could try reverting the extension to `0.12.2` until #123 is resolved.

**Comment by tamasfe** [comment]
This should be fixed in taplo `0.6.1`, and vscode extension version `0.13.1`.
